OpenKeychain/build.gradle

```diff
@@ -207,10 +213,17 @@ android {
         }
 
         debug {
-            // disable ProGuard for faster compile times for debug builds
-            minifyEnabled false
+            minifyEnabled true
+
+            proguardFiles = buildTypes.release.proguardFiles
 
-            multiDexEnabled true
+            testProguardFiles getDefaultProguardFile('proguard-android.txt'),
```

Comment:
``testProguardFiles`` only needs the additional files, i.e., ``'proguard-rules-test.pro'``

https://github.com/googlesamples/android-testing-templates/tree/master/AndroidTestingBlueprint#proguard

Reply:
I did this here: https://github.com/open-keychain/open-keychain/tree/hagau-proguard

Reply:
Thanks, I'm sure I have read that page at some point, but forgot to clean up that part of the patch.